docs/maintenance.md

```
@@ -11,7 +11,7 @@ echo "🔍 Vérification quotidienne du portfolio..."
 
 # 1. Statut du site
 echo "📡 Vérification du statut..."
-curl -s -o /dev/null -w "%{http_code}" https://kevine.dev
+curl -s -o /dev/null -w "%{http_code}" https://kevine-portfolio.vercel.app/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)